components/insight/SRC/org/openmicroscopy/shoola/util/ui/lens/LensModel.java

```diff
@@ -76,7 +76,7 @@
 	 * The default size of the pre-allocated buffer used to store the 
 	 * zoomed image. 
 	 */
-	final static int		DEFAULT_SIZE = 562500;
+	final static int		DEFAULT_SIZE = 750000;
```

Comment:
Got this number from the exception, which said the buffer has to be greater than 749999 (when zoom is set to 1000%). Maybe it can be calculated, can do some more investigation.